src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/content/countycourtjudgment/AmountRemainingContentProvider.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.hmcts.cmc.claimstore.services.staff.content.countycourtjudgment;
+
+import uk.gov.hmcts.cmc.claimstore.models.Claim;
+import uk.gov.hmcts.cmc.claimstore.models.Interest;
+import uk.gov.hmcts.cmc.claimstore.models.InterestDate;
+import uk.gov.hmcts.cmc.claimstore.models.amount.AmountBreakDown;
+import uk.gov.hmcts.cmc.claimstore.services.interest.InterestCalculationService;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+
+import static uk.gov.hmcts.cmc.claimstore.utils.Formatting.formatMoney;
+
+public final class AmountRemainingContentProvider {
+
+    private AmountRemainingContentProvider() {
+        // Utils class no constructing
+    }
+
+    public static String calculate(
```

Comment:
I do not like the fact `calculate` method returns string, that feels unexpected to me. I think that class should do the math to calculate remaining amount, since that be called `RemainingAmountCalculator` and return BigDecimal amount.

Then amount would be just retrieved and formatted in CCJ content provider.